index.html

```diff
@@ -4,9 +4,320 @@
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="about.html">О компании</a>
+          </li>
+        </ul>
+        <ul class="navigation-list navigation-user">
+          <li class="navigation-item ">
+            <a class="navigation-link navigation-telephone" href="#">+7 800 555-86-28</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">
+              <span class="visually-hidden">
+                Поиск
+              </span>
+            </a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">Вход</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link navigation-basket" href="#">
+              <span class="visually-hidden">
+                Корзина
+              </span>
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
 
+    <main class="main-index">
+      <h1 class="visually-hidden">Магазин мороженого Gllacy Shop</h1>
+      <section class="main-gallery">
+        <ul class="slider">
+          <li>
+            <h2 class="main-gallery-title">Нежный пломбир с клубничным джемом</h2>
+            <span class="basic-text">Натуральное мороженое из свежих сливок и молока с вкуснейшим клубничным джемом – это идеальный десерт для всей семьи.</span>
+            <button class="button" type="submit">Заказать</button>
+          </li>
+          <li>
+            <img class="slider-image" src="#" alt="Нежный пломбир с клубничным джемом">
+            <button class="slider-button slider-prev" type="button">
+              <span class="visually-hidden">Предыдущее мороженое</span>
+            </button>
+            <button class="slider-button slider-next" type="button">
+              <span class="visually-hidden">Следующее мороженое</span>
+            </button>
+          </li>
+          <li class="slider-image-preview">
+            <a href="#">
+              <span class="visually-hidden">
+                Сливочное мороженое со вкусом банана
+              </span>
+            </a>
+          </li>
+          <li class="slider-image-preview">
+            <a href="#">
+              <span class="visually-hidden">
+                Карамельный пломбир с маршмеллоу
+              </span>
+            </a>
+          </li>
+        </ul>
+        <ol class="slider-pagination">
+          <li class="slider-pagination-item">
+            <button class="slider-pagination-button slider-pagination-button-current" type="button">
+              <span class="visually-hidden">Нежный пломбир с клубничным джемом</span>
+            </button>
+          </li>
+          <li class="slider-pagination-item">
+            <button class="slider-pagination-button" type="button">
+              <span class="visually-hidden"> Сливочное мороженое со вкусом банана</span>
```

Comment:
Опять пробел заблудился
```suggestion
              <span class="visually-hidden">Сливочное мороженое со вкусом банана</span>
```

Reply:
Поправила